### PR TITLE
Added responsiveness in css files

### LINKED ISSUE #7674: Responsiveness in CSS
### Description

Details about what is a responsive website and how it's implemented using CSS 

### Domain

Frontend Dev HTML/CSS/JS

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by niharikamahajan02**
<hr>

## Description 📜

Details about what is a responsive website and how it's implemented using CSS
Fixes #7674 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [x] Frontend Dev HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by niharikamahajan02** [comment]
please review

File: Web_Development/FrontEnd/CSS_REPONSIVENESS/reponsiveness.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+</style>
+</head>
+<body>
+
+<p>Resize the browser window. In landscape its background color is "lightblue", otherwise it is "lightgreen".</p>
+
+</body>
+</html>
+
+```
```
Comment (code-reaper08) [inline Web_Development/FrontEnd/CSS_REPONSIVENESS/reponsiveness.md:38]: Wrong image captions and example taken from https://www.w3schools.com/css/tryit.asp?filename=tryresponsive_mediaquery_orientation2

Please try to do examples on your own, or else at least specify the source.

File: Web_Development/FrontEnd/CSS_REPONSIVENESS/reponsiveness.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+}
+</style>
+</head>
+<body>
+<h2>Font size change</h2>
+<div class="ele">This will change the font</div>
+<p>When the browser's width is 350px wide or less, set the font-size to 40px. When it is 351px or wider, set the font-size to 20px and change text color to green. </p>
+</body>
+</html>
+```
```
Comment (code-reaper08) [inline Web_Development/FrontEnd/CSS_REPONSIVENESS/reponsiveness.md:98]: Please add source https://www.w3schools.com/css/tryit.asp?filename=trycss_mediaqueries_fontsize

File: Web_Development/FrontEnd/CSS_REPONSIVENESS/reponsiveness.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+}
+</style>
+</head>
+<body>
+<h2>element hiding</h2>
+<div class="ele">This is to be hidden</div>
+<p>When the browser's width is 600px wide or less, hide the element. Resize the browser window to see the effect.</p>
+</body>
+</html>
+```
```
Comment (code-reaper08) [inline Web_Development/FrontEnd/CSS_REPONSIVENESS/reponsiveness.md:132]: Please add source https://www.w3schools.com/css/tryit.asp?filename=tryresponsive_mediaquery_orientation2

File: Web_Development/FrontEnd/CSS_REPONSIVENESS/reponsiveness.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
+# RESPONSIVENESS IN CSS
+
+ As we all know webpages can be viewed on many devices like desktops, tablets, and phones. For web pages to look attractive and eye-catching on each device , responsiveness should be used . Without this our website may look interactive in some dimensions or on some devices like laptops but may look undesirable and unorganized in other devices like phones, tablets etc.
+
+> We  use CSS and HTML to resize, hide, shrink, enlarge, or move the content to make it look good on any screen.
```
Comment (code-reaper08) [inline Web_Development/FrontEnd/CSS_REPONSIVENESS/reponsiveness.md:5]: Remove formatting

**Review by code-reaper08 — CHANGES_REQUESTED** [review]
Hello @niharikamahajan02 
I've left a review, please check.
I would also suggest you to add images as upload assets (using drag and drop)
This would make the repo much more light weight.

If you need any help tag me here :-)

**Comment by niharikamahajan02** [comment]
@code-reaper08 , please review again , I have done the suggested changes .
Also have uploaded the images again , please tell if this was the way to upload images , you told me about.

**Review by code-reaper08 — APPROVED** [review]
LGTM ✨

**Comment by code-reaper08** [comment]
> @code-reaper08 , please review again , I have done the suggested changes . Also have uploaded the images again , please tell if this was the way to upload images , you told me about.

Yep Images are good now!
Thanks for contributing 😄

**Comment by niharikamahajan02** [comment]
> > @code-reaper08 , please review again , I have done the suggested changes . Also have uploaded the images again , please tell if this was the way to upload images , you told me about.
> 
> Yep Images are good now! Thanks for contributing 😄

Thankss!!

**Pull request merged**